UML/TapiPhotonicMedia.uml

```diff
@@ -583,20 +560,19 @@ For FLEX grid type, the adjusmentGranularity is 0.00625 THz and the slot width i
           <ownedComment xmi:type="uml:Comment" xmi:id="_QuSUsI6UEeeyZasfnNSonw" annotatedElement="_KjQ3xRKOEeajhbtskMXJfw">
             <body>Specifies the frequency grid standard used to determine the nominal central frequency and frequency slot width</body>
           </ownedComment>
-          <defaultValue xmi:type="uml:InstanceValue" xmi:id="_nJd3wHk1EeiO-c_khjKlFQ" instance="_G--TMHPEEeiPPoPDo3q5jA"/>
+          <defaultValue xmi:type="uml:InstanceValue" xmi:id="_nJd3wHk1EeiO-c_khjKlFQ" type="_iHIbYI6KEeeyZasfnNSonw" instance="_G--TMHPEEeiPPoPDo3q5jA"/>
         </ownedAttribute>
         <ownedAttribute xmi:type="uml:Property" xmi:id="_87l7YI6VEeeyZasfnNSonw" name="adjustmentGranularity" type="_htJmYI6QEeeyZasfnNSonw">
           <ownedComment xmi:type="uml:Comment" xmi:id="_XbMmsI6WEeeyZasfnNSonw" annotatedElement="_87l7YI6VEeeyZasfnNSonw">
             <body>Adjustment granularity in Gigahertz. As per ITU-T G.694.1, it is used to calculate nominal central frequency (in THz)</body>

```

Comment:
AS far as I understand - there is an agreement to use MHz as unit for frequencies. > should to be updated in steriotypes Units and comments.